masterlist.yaml

```diff
@@ -8605,7 +8605,7 @@ plugins:
   - name: 'CALLOFTHENORTH_COMPLEMENT.esp'
     msg: [ *obsolete ]
     url:
-      - http://www.nexusmods.com/skyrim/mods/29040
+      - https://www.nexusmods.com/skyrim/mods/29670
```

Comment:
You could inline that location. It's not critical though and can be merged as it is.
```yaml
  - name: 'CALLOFTHENORTH_COMPLEMENT.esp'
    msg: [ *obsolete ]
    url: [ 'https://www.nexusmods.com/skyrim/mods/29670' ]
```

Reply:
It's a good suggestion. Done as of ecdd10b